mariadb_copy.md

```diff
@@ -58,6 +58,37 @@ DB_ROOT_PASSWORD=SomePassword
       mysql -h "$PODIFIED_MARIADB_IP" -uroot "-p$DB_ROOT_PASSWORD" -e 'SHOW databases;'
   ```
 
+## Procedure - stopping control plane services
+
+From each controller node it is necessary to stop the
+control-plane services to avoid inconsistencies in the
+data migrated for the data-plane adoption procedure.
+
+1- Log in to a controller node with the CLI enabled.
+
+2- Stop the services.
+
+```bash
+ServicesToStop=("tripleo_horizon.service"
+                "tripleo_keystone.service"
+                "tripleo_cinder_api.service"
+                "tripleo_glance_api.service"
+                "tripleo_neutron_api.service"
+                "tripleo_nova_api.service"
+                "tripleo_placement_api.service")
+
+echo "Stopping the OpenStack services"
+
+for service in ${ServicesToStop[*]}; do
+    echo "Stopping the service: $service"
+    systemctl stop $service
+done
+```
```

Comment:
I think it would be a bit nicer if we could assume the snippet is executed from outside the controller, and the snippet would include ssh command. E.g. something like

CONTROLLER1_SSH="ssh -F $ENV_DIR/director_standalone/vagrant_ssh_config vagrant@standalone"
CONTROLLER2_SSH=":"
CONTROLLER3_SSH=":"

And in the for cycle we could have

```
$CONTROLLER1_SSH sudo systemctl stop $service
$CONTROLLER2_SSH sudo systemctl stop $service
$CONTROLLER3_SSH sudo systemctl stop $service
```

This could cover both 1 controller and 3 controller scenarios. The ":" in the SSH commands is a bash no-op. Not a blocker but i think it would make it a bit nicer, allow people to make their own copy-pastable snippets tailored for their environment. WDYT? Maybe a follow-up PR?

Reply:
cool, done

Reply:
:+1: thanks!